docs/general/ddl/analyzers.rst

```diff
@@ -110,6 +110,15 @@ This is the same as the `standard-analyzer`_ analyzer.
 
 Uses the :ref:`lowercase-tokenizer` tokenizer.
 
+.. _plain-analyzer:
+
+``plain``
+----------
+
+``type='plain'``
+
+Creates one single token without any further analysis from the field-contents.
```

Comment:
```suggestion
This is the same as the `keyword`_ analyzer.
```

Reply:
`plain` is a special CrateDB analyzer alias, I'm not sure if we allow to extend from it when creating custom analyzer.
Can you please verify that and note it here if not extendable (note to extend from `keyword`_ analyzer instead)?

Reply:
Good point, thank you, will clarify that.

Reply:
I am actually not quite sure if it would be better to drop the plain analyzer completely and simply reference for `plain` to the keyword analyzer.

Reply:
@mechanomi what is your opinion here ?

Reply:
I think it makes sense to keep it in, and explain that it's an alias

Reply:
👍